producer/callback.go

```diff
@@ -39,6 +48,59 @@ func HandleSmPolicyNotify(request *httpwrapper.Request) *httpwrapper.Response {
 	return httpwrapper.NewResponse(http.StatusNotImplemented, nil, nil)
 }
 
-// TODO: handle SM Policy Notify
+// SmPolicyNotifyProcedure TODO: handle SM Policy Notify
 func SmPolicyNotifyProcedure(supi string, notification models.PolicyDataChangeNotification) {
 }
+
+func HandleNfSubscriptionStatusNotify(request *httpwrapper.Request) *httpwrapper.Response {
```

Comment:
Should we add a comment to the function for documentation purposes?

Reply:
Comment is added.